_posts/0000-01-02-JorgeBaquerizo-CS.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![Image of Biden](https://www.aljazeera.com/wp-content/uploads/2020/11/2020-11-07T040119Z_1284520795_RC24YJ9HZK65_RTRMADP_3_USA-ELECTION-BIDEN.jpg?resize=770%2C513)

**Foto de Biden**